templates/zerver/help/restrict-name-and-email-changes.md

```diff
@@ -15,19 +15,20 @@ source, and synced into Zulip via the [Zulip API](/api) or another method.
 
 {settings_tab|organization-permissions}
 
-2. Under the **User identity**, select **Prevent users from changing their name**.
+2. Under **User identity**, select **Prevent users from changing their name**.
 
 {!save-changes.md!}
 
 {end_tabs}
 
-## Prevent all email changes
+## Restrict email changes
 
-By default, any user can
-[change their email address](/help/change-your-email-address). However, you
-can instead prevent all email changes. This is especially useful for
-organizations that are using LDAP or another single
-sign-on solution to manage user emails.
+By default, any user can [change their email address][change-email].
+However, you can instead prevent users from changing their email address.
+Organization administrators can always change their own email address.
```

Comment:
What if instead we do something like this?

!!! tip ""

    Administrators can always change any user's email address, including their own.

Reply:
I don't think that administrators can change other users' email addresses (whether the setting is enabled or not), so the tip text would just be that administrators can always change their own email.

The same goes for avatars / profile pictures. Only the user can change their own avatar, and the ability to do so is never restricted for owner / administrator roles.

Reply:
Here are screenshots of the tip version of these two articles. The tips could also go after the instruction blocks ...

<details>
<summary>Restrict name and email changes</summary>

![Screenshot from 2023-01-04 15-31-17](https://user-images.githubusercontent.com/63245456/210577491-529786a6-5a5c-45ad-afee-867b6548c0e3.png)
</details>
<details>
<summary>Restrict profile picture changes</summary>

![Screenshot from 2023-01-04 15-26-30](https://user-images.githubusercontent.com/63245456/210576677-c64d9dd2-c3f9-4b1e-b2e5-d3da41dec8e9.png)
</details>

Reply:
Oops, I should have checked how this works -- sorry about that!

 Anyway, I'm happy with the proposed tip version. Can you mark the PR for integration review once it's been updated? Thanks!